.travis.yml

```diff
@@ -42,6 +42,9 @@ matrix:
     - os: linux
       python: 3.5
       env: QT=PyQt5 WITH_PYAMG=1 MINIMUM_REQUIREMENTS=1
+    - os: linux
```

Comment:
Can't we rather add this flag to one of the existing matrix builds?  We have to be careful here, because we only have a fixed number of parallel Travis-CI builds available.

Reply:
Oh yea, I was suggesting we add it to the one on line 44 just above it.
Good catch.

Reply:
just like this ?